src/misclibs.F

```diff
@@ -3037,10 +3037,13 @@ subroutine calccfl_fold(nstat,rstat,dt,acfl,uh,vh,mh,ua,va,wa,writeit)
       dtdy=0.5*dt*rdy
       dtdz=0.5*dt*rdz
 
-      cfl = -1.0
-      imaxt = 0
-      jmaxt = 0
-      kmaxt = 0
+      !$acc parallel loop gang vector default(present)
+      do index=1,ni*nj
+        cfl(index) = -99999.0
+        imaxt(index) = 0
+        jmaxt(index) = 0
+        kmaxt(index) = 0
+      enddo
```

Comment:
According to @johnmauff and @george-bryan , these calculations were not done correctly on GPU before and led to a CFL violation error later. The changes here fix this CFL error.